docs/source/configuration/internalproxy.md

```diff
@@ -1,28 +1,42 @@
-# Internal proxy to API
+# Internal proxy to content backend API
+
+The server side Volto SSR process (based on Razzle) has an internal proxy to the backend API
+enabled by default.
 
-While in development, Volto has an internal proxy to the backend API enabled by default.
 It provides a better developer experience out of the box, so the developer doesn't has to
 deal with [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) and can focus on
 develop/test drive/demo Volto.
 
+To understand the need for the internal proxy, there are three processes running in a Volto website:
+
+1. A frontend web application running in your browser (Javascript)
+2. A Node.js server process that delivers the javascript to the client and does
+   Server Side Generation (SSR) of your pages on first request (Javascript, the
```

Comment:
Watch out, SSG is another thing (what NextJS does). So:

"...does Server Side Rendering (SSR)..."

Reply:
fixed